hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -2900,37 +2900,6 @@ public void testHotRestartEncryptionAtRest_whenVault() {
         assertEquals("TLS", sslConfig.getProperties().get("protocol"));
     }
 
-    @Override
-    @Test
-    public void testMapEvictionPolicyClassName() {
-        String mapEvictionPolicyClassName = "com.hazelcast.map.eviction.LRUEvictionPolicy";
-        String xml = HAZELCAST_START_TAG
-                + "<map name=\"test\">"
-                + "<map-eviction-policy-class-name>" + mapEvictionPolicyClassName + "</map-eviction-policy-class-name> "
```

Comment:
is there another test covering map eviction configured with custom comparator class name?

Reply:
added a specific test for this case.